src/components/ServiceRequestForm.jsx

```diff
@@ -333,16 +363,15 @@ const ServiceRequestForm = (props, errors, touched) => {
 
 				<RequestSubCategory
 					requestType={requestType}
-					subRequestType={subRequestType}
+					subRequestType={subRequestType }
 					errorsSubRequestType={localProps.errors.subRequestType}
 					touchedSubRequestType={localProps.touched.subRequestType}
 					pageFieldName={RequestPage.SubCategoryLabel}
 					handleServiceSubRequestChange={handleServiceSubRequestChange}
 					rest={rest}
 					subCategories={subCategories} />
 
-
-				{localProps.values.shouldDisableForm && notes}
+				{localProps.values.shouldDisableForm === true && notes}
```

Comment:
this can probably be simlified, to just `localProps.values.shouldDisableForm === true` but it okay if it stays for now.